### PR TITLE
Allow adjusting Ip refresh timeout

### PR DESCRIPTION
**Pull request by sargun**

### CODE REVIEW

File: executor/runtime/docker/docker.go [hunk 1]
```diff
@@ -150,6 +151,11 @@ func NewConfig() (*Config, []cli.Flag) {
 			Value:       time.Minute * 1,
 			Destination: &cfg.waitForSecurityGroupLockTimeout,
 		},
+		cli.DurationFlag{
+			Name:        "titus.exwecutor.networking.ipRefreshTimeout",
```
Comment (andrew-leung) [inline executor/runtime/docker/docker.go]: Typo?

File: vpc/allocate/ip_pool_manager.go [hunk 2]
```diff
@@ -59,7 +59,8 @@ func (mgr *IPPoolManager) assignMoreIPs(ctx *context.VPCContext, batchSize int)
 	}
 
 	originalIPCount := len(mgr.networkInterface.IPv4Addresses)
-	for i := 0; i < 10; i++ {
+	now := time.Now()
+	for time.Since(now) < ipRefreshTimeout {
```
Comment (andrew-leung) [inline vpc/allocate/ip_pool_manager.go]: In the comment on line 69, aren't we not retrying in that case?

File: executor/runtime/docker/docker.go (hunk 1 above)
Comment (sargun) [reply executor/runtime/docker/docker.go]: Typo.

File: vpc/allocate/ip_pool_manager.go (hunk 2 above)
Comment (sargun) [reply vpc/allocate/ip_pool_manager.go]: Er, that means we retry to "lease" or "allocate" an IP from from the IPs assigned to this ENI, whereas this is to "assign" IPs. I can change the comment if it helps?

**Pull request merged**